src/KM_Controls.pas

```diff
@@ -142,6 +137,11 @@   TKMControl = class
     function GetSelfHeight: Integer; virtual;
     function GetSelfWidth: Integer; virtual;
     procedure UpdateVisibility; virtual;
+    procedure ControlMouseDown(Sender: TObject; Shift: TShiftState); virtual;
+    procedure ControlMouseUp(Sender: TObject; Shift: TShiftState); virtual;

```

Comment:
8.Removed unnessesery events from TKMControl fOnControlMouseDown/Up. Must use ControlMouseDown/Up methods hierarhy instead.
Detailed description above.
Moved procedure ControlMouseDown/Up from public section into protected one to remove external use of this two procedures.